src/main/webapp/i18n/en/feedback.json

```
@@ -9,7 +9,7 @@
             "updated": "Updated Feedback with identifier {{ param }}",
             "deleted": "Deleted Feedback with identifier {{ param }}",
             "delete": {
-                "question": "Are you sure you want to delete Feedback {{ id }}?"
+                "question": "Are you sure you want to delete the Feedback \"{{ text }}\"?"
```

```suggestion
                "question": "Are you sure you want to delete the feedback \"{{ text }}\"?"
```
I'm also thinking about adding a colon, what is your opinion?
```suggestion
                "question": "Are you sure you want to delete the feedback: \"{{ text }}\"?"
```